### DISCUSSION TITLE
Dropdown fields not available

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16571] [How to] [answered]
Hello, Please, I need confirmation because suddenly all the drop-down list fields come out as if they did not exist in the database.

![image](https://user-images.githubusercontent.com/96985476/194951779-53a8a977-b10d-4f56-9c49-b25e0c20754a.png)

![image](https://user-images.githubusercontent.com/96985476/194952331-6295f9d4-aa4c-4747-8245-a3a9b50dd5d5.png)

![image](https://user-images.githubusercontent.com/96985476/194951920-8a46f8a7-3655-43c8-ad20-9d42d617cb6f.png)

**Comment by rskrzypczak** [comment, marked as answer]
Check if the set values are present in the picklist (case sensitive).
And verify that your role is authorized to these picklist values.
![image](https://user-images.githubusercontent.com/10221999/194996279-a595e513-53cb-4600-885a-5e588c485b2f.png)

**Reply by krseguros** [reply]
Thank you, I validated and effectively assigned the fields to the corresponding roles, after this they can be seen and edited, however, a few days ago this was not the case, by default all users including that I am an administrator could see and edit all dropdown list fields.

Now from what I see I have to assign all the list fields to the corresponding roles one by one so that they can be edited.

Is there any way to make it massive "ALL" and not one by one?

![image](https://user-images.githubusercontent.com/96985476/195079214-5f77fe9a-5f67-4c45-a51b-90ccbdc42979.png)

![image](https://user-images.githubusercontent.com/96985476/195079068-22585291-8687-4755-926d-9d5fe96b4833.png)

![image](https://user-images.githubusercontent.com/96985476/195079328-3e46dbab-2fa4-4a6e-a703-dc77ec940fdc.png)
![image](https://user-images.githubusercontent.com/96985476/195079527-f5edb848-9170-4527-adaa-4808679dcb51.png)

**Reply by rskrzypczak** [reply]
> Is there any way to make it massive "ALL" and not one by one?

Unfortunately, no.
Overall, I recommend upgrading your system to a newer version to avoid such cases in the future.

**Reply by krseguros** [reply]
It's updated
![image](https://user-images.githubusercontent.com/96985476/195088238-1ab634a1-af7d-4f41-81f2-afa77c4d83ba.png)
